Modules/clinic/_testclinic_limited.c.h

```diff
@@ -37,8 +37,7 @@ my_int_func(PyObject *module, PyObject *arg_)
     int arg;
     int _return_value;
 
-    arg = PyLong_AsInt(arg_);
-    if (arg == -1 && PyErr_Occurred()) {
+    if (!PyArg_Parse(arg_, "i:my_int_func", &arg)) {
```

Comment:
It looks like regression, but actually not all inlined parsing code is compatible with the limited C API. In following PR I will implement inlining correctly, only for converters compatible with the limited C API. It will be large but mostly boring change (adding new parameter and simple checks in tens of functions), so I did not include it here.